README.md

```diff
@@ -48,9 +48,11 @@ Additionally, the following custom functions are supported:
 - `fromConfigMap` returns the value of a key inside a `ConfigMap`. For example,
   `{{ fromConfigMap "namespace" "config-map-name" "key" }}`.
 - `fromSecret` returns the value of a key inside a `Secret`. For example,
-  `{{ fromSecret "namespace" "secret-name" "key" }}`.
+  `{{ fromSecret "namespace" "secret-name" "key" }}`. If the `protect` template function is enabled,
```

Comment:
I think `fromSecret` always encrypt. But the description makes me think `protect` needs to be used with `fromSecret`

Reply:
You're right. I need to reword that to it being always encrypted if the encryption mode is set to `EncryptionEnabled`.